src/lua/MapApi.cpp

```diff
@@ -778,7 +778,7 @@ int LuaContext::l_create_block(lua_State* l) {
     EntityData& data = *(static_cast<EntityData*>(lua_touserdata(l, 2)));
```

Comment:
Is this sending a lua nil to a value of two? If so, it should be changed to `EntityData& data = *(static_cast<EntityData*>(lua_touserdata(l, 2)));`